backend/prisma/schema.prisma

```diff
@@ -73,4 +73,5 @@ model Transactions {
   savings_account       Float?
   time                  DateTime @default(now()) // column to store the timestamp at which a trasaction was logged so we can fetch most recent
   @@index([time])
+  transaction_date      Int? // this stores a basic int to test over time, entered manually
```

Comment:
This is purely for debugging/testing?

Reply:
I think this could go either way. Seeing a date would probably make more sense than an internal metric like weeks since joining which is effectively what the int value captures. But I'm also not sure how insightful connecting expenditure to a date is over the more important component of seeing the change over time more broadly. In short, changing to date is something that'd be a nice feature to change to but it isn't necessary